air/src/execution_step/value_types/canon_stream.rs

```diff
@@ -48,6 +48,10 @@ impl CanonStream {
         }
     }
 
+    pub(crate) fn from_values_and_tetraplet(values: Vec<ValueAggregate>, tetraplet: Rc<SecurityTetraplet>) -> Self {
```

Comment:
Is it different from `CanonStream::new`?

Reply:
No. Fixed